advanced_source/static_quantization_tutorial.rst

```diff
@@ -0,0 +1,665 @@
+`these lines <https://github.com/pytorch/tutorials/blob/master/Makefile#L97-L98>`_  
+from the `Makefile <https://github.com/pytorch/tutorials/blob/master/Makefile>`_. 
+
+To run the code in this tutorial using the entire ImageNet dataset, on the other hand, you could download 
+the data using ``torchvision`` following  
+`here <https://pytorch.org/docs/stable/torchvision/datasets.html#imagenet>`_. For example,  
+to download the training set and apply some standard transformations to it, you could use:  
+
+.. code:: python  
+
+    import torchvision  
+    import torchvision.transforms as transforms 
+
+    imagenet_dataset = torchvision.datasets.ImageNet( 
+        '~/.data/imagenet', 
+        split='train',  
+        download=True,  
+        transforms.Compose([  
+            transforms.RandomResizedCrop(224),  
+            transforms.RandomHorizontalFlip(),  
+            transforms.ToTensor(),  
+            transforms.Normalize(mean=[0.485, 0.456, 0.406],  
+                                 std=[0.229, 0.224, 0.225]),  
+        ])  
+
+With the data downloaded, we show functions below that define dataloaders we'll use to read 
+in this data. These functions mostly come from  
+`here <https://github.com/pytorch/vision/blob/master/references/detection/train.py>`_.
+
+
+.. code:: python
+
+    def prepare_data_loaders(data_path):  
+
+        traindir = os.path.join(data_path, 'train') 
+        valdir = os.path.join(data_path, 'val') 
+        normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],  
+                                         std=[0.229, 0.224, 0.225]) 
+
+        dataset = torchvision.datasets.ImageFolder( 
+            traindir, 
+            transforms.Compose([  
+                transforms.RandomResizedCrop(224),  
+                transforms.RandomHorizontalFlip(),  
+                transforms.ToTensor(),  
+                normalize,  
+            ])) 
+
+        dataset_test = torchvision.datasets.ImageFolder(  
+            valdir, 
+            transforms.Compose([  
+                transforms.Resize(256), 
+                transforms.CenterCrop(224), 
+                transforms.ToTensor(),  
+                normalize,  
+            ])) 
+
+        train_sampler = torch.utils.data.RandomSampler(dataset) 
+        test_sampler = torch.utils.data.SequentialSampler(dataset_test) 
+
+        data_loader = torch.utils.data.DataLoader(  
+            dataset, batch_size=train_batch_size, 
+            sampler=train_sampler)  
+
+        data_loader_test = torch.utils.data.DataLoader( 
+            dataset_test, batch_size=eval_batch_size, 
+            sampler=test_sampler) 
+
+        return data_loader, data_loader_test  
+
+
+Next, we'll load in the pre-trained MobileNetV2 model. We provide the URL to download the data from in ``torchvision``  
+`here <https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenet.py#L9>`_. 
+
+.. code:: python
+
+    data_path = 'data/imagenet_1k'  
```

Comment:
Lets replace this from:
data_path = 'data/imagenet_1k'
to
data_path = '~/.data/imagenet'